### ISSUE TITLE
Flask Frameworks - Remove no longer maintained frameworks 

### ISSUE THREAD

**Issue by dinkopehar** [humiaozuzu/awesome-flask#198]
I was reading about some of the Flask frameworks mentioned in this repository under Frameworks permalink.

I noticed that there is `Flask-RESTPlus` and `Flask-restless`.

If you go to Flask-Restless [repo](https://github.com/jfinkels/flask-restless), you can see that it states:

> NO LONGER MAINTAINED
> This repository is no longer maintained due to lack of time.

Also, for Flask-RESTPlus, in first sections in [repo](https://github.com/noirbizarre/flask-restplus#flask-restplus) this is written:

> This project has been forked to Flask-RESTX and will be maintained by by the python-restx organization. Flask-RESTPlus should be considered unmaintained.

> The community has decided to fork the project due to lack of response from the original author @noirbizarre. We have been discussing this eventuality for almost a year.

> Things evolved a bit since that discussion and a few of us have been granted maintainers access to the github project, but only the original author has access rights on the PyPi project. As such, we been unable to make any actual releases. To prevent this project from dying out, we have forked it to continue development and to support our users.

---

So in conclusion, consider adding **DEPRECATED** around these two or **NOT MAINTAINED** and adding `Flask-RESTX` since it's one being developed.

**Comment by EtiennePelletier** [comment]
There's already a PR for Flask-RESTPlus --> Flask-RESTX: https://github.com/humiaozuzu/awesome-flask/pull/197.

**Comment by dinkopehar** [comment]
Nice. Maybe add deprecated tag around Flask Restless in your pull request. Didn't see your pull request, just opened an issue as recommendation

**Comment by EtiennePelletier** [comment]
Anyone can open a new PR to replace Flask Restless with https://github.com/mrevutskyi/flask-restless-ng :smile: I would suggest doing all these changes in separate PRs so that they can be merged independantly.
